manifests/m/mmanela/markdownoutlook/1.0.0/mmanela.markdownoutlook.yaml

```
@@ -1,7 +1,7 @@
 PackageIdentifier: mmanela.markdownoutlook
-PackageVersion: 1.2.0
-PackageName: MarkdownOutlook
-Publisher: mmanela
+PackageVersion: 1.0.0
```

```suggestion
PackageVersion: 1.0.0
```
This change was done to match Control Panel to prevent upgrade loops due to version mismatch.